src/language-js/parser-meriyah.js

```diff
@@ -28,17 +28,17 @@ const parseOptions = {
   // Enable non-standard parenthesized expression node
   preserveParens: false,
   // Enable lexical binding and scope tracking
-  lexical: true,
```

Comment:
Why turn off this option? By default all parsers have it "build-in". For example for cases like `"use strict"; function x(a, [a]) {}` it should throw an error. The error will not be thrown if this is set to `false`.

Reply:
We don't need throw. There are many invalid cases we can format.